docs/source/tutorial/add_another_pipeline.md

```diff
@@ -456,9 +437,9 @@ After executing `kedro run`, you should see output as follows:
                     INFO     Completed 9 out of 9 tasks                                                            sequential_runner.py:85
                     INFO     Pipeline execution completed successfully.
 ```
-</details>
+</details> <br />
 
-#### How it works: the modular `pipeline()` wrapper
+### How it works: the modular `pipeline()` wrapper
```

Comment:
This still contains a casual mention to `kedro-viz` inside a seemingly optional "how it works" section, wondering if we could introduce it more explicitly at an earlier stage as a subsection of its own (in this page, or maybe the previous one about the data engineering pipeline)